app/components/validated-input.js

```diff
@@ -0,0 +1,44 @@
+// Copied from ember-cpi-validations/tests/dummy/app - should eventually be part of ember-osf but getting errors with dependencies
+
+// BEGIN-SNIPPET validated-input
```

Comment:
@abought I am not sure I understand.   I don't think this is part of the ember-cp-validations input, I borrowed this from the ember-cp-validations dummy app.  So did max, he also copied the whole thing out.  Max is doing almost the exact same thing I'm doing - ours our both working.  We are going to have very similar validated-input components though so need to come up with a way to consolidate.